app/src/main/java/com/eventyay/organizer/core/event/create/CreateEventViewModel.java

```diff
@@ -250,4 +260,17 @@ public void updateEvent() {
                 }, Logger::logError));
     }
 
+    //Method for storing user uploaded image in temporary location
+    public void uploadImage(ImageData imageData) {
+        compositeDisposable.add(
+            eventRepository
+                .uploadEventImage(imageData)
+                .doOnSubscribe(disposable -> progress.setValue(true))
+                .doFinally(() -> progress.setValue(false))
+                .subscribe(uploadedImage -> {
+                    onSuccess.setValue("Image Uploaded Successfully");
+                    imageUrl = uploadedImage;
+                    getImageUrlLiveData();
```

Comment:
Set value here

Reply:
Still not solved

Reply:
Remove this line. Why are we getting the livedata here?

Reply:
Sorry, it's removed now.